theano/sandbox/solve.py

```diff
@@ -1,75 +0,0 @@
-from __future__ import absolute_import, print_function, division
```

Comment:
Instead of deleting sandbox.solve, we should keep the file as a placeholder that raises and error that tells users to import slinalg.solve.